tests/js/testUtils.js

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2017 OCAD University
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/waharnum/sjrk-storyTelling/master/LICENSE.txt
+*/
+
+/* global fluid, sjrk */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    sjrk.storyTelling.testUtils = {};
```

Comment:
Can we use `fluid.registerNameSpace` (http://docs.fluidproject.org/infusion/development/CoreAPI.html#fluid-registernamespace-path-) here instead to make sure the namespace exists?